src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.test.tsx

```diff
@@ -210,7 +305,7 @@ it("shows the 'Start a free scan' CTA to free US-based users who haven't perform
   );
   render(<ComposedDashboard />);
 
-  const freeScanCta = screen.getByRole("link", { name: "Start a free scan" });
+  const freeScanCta = screen.queryByRole("link", { name: "Start a free scan" });
```

Comment:
Heh, I'd expect changes the other way around - `getByRole` if you need to check if something exists, `queryByRole` if you're verifying that it _doesn't_ exist. See the docs [here](https://testing-library.com/docs/queries/about/#types-of-queries):

> **queryBy...** (...) This is useful for asserting an element that is not present.

The reason `getBy` is useful is that it throws a descriptive error if it doesn't match, telilng you what other elements of the given role it _did_ find, for example.

Reply:
Thanks! Changed in [b05f5ac](https://github.com/mozilla/blurts-server/pull/3494/commits/b05f5ac82a4e5ec2d0a2319ad92b8690b64a0599).